README.md

```diff
@@ -63,9 +65,9 @@ Your Answer:
 argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
-```js
-// write code here
-```
+function doSomething (x,y) {
+    return thingToDo(x) + (y);
+}
 
```

Comment:
This would error out immediately, as the `(x) + (y)` syntax is not valid javascript. Beyond that, this solution doesn't do everything that the question asks for. **Please** review the notes from the Callbacks lesson: https://github.com/ga-dc/curriculum/tree/master/02-intermediate-front-end/events-and-callbacks